libmscore/text.cpp

```diff
@@ -1050,13 +1050,7 @@ QRectF Text::cursorRect() const
 
 QColor Text::textColor() const
       {
-      if (score() && !score()->printing()) {
-            if (selected())
-                  return (track() > -1) ? MScore::selectColor[voice()] : MScore::selectColor[0];
-            if (!visible())
-                  return Qt::gray;
-            }
-      return textStyle().foregroundColor();
+      return curColor();
```

Comment:
Hmm, as per https://github.com/musescore/MuseScore/pull/3388/commits/bb8d93f931c17101770d990fd5d556245e2017d2#r161251176 this doesn't work?